velox/core/PlanNode.cpp

```diff
@@ -1557,6 +1557,28 @@ PlanNodePtr TableWriteNode::create(const folly::dynamic& obj, void* context) {
       source);
 }
 
+void TableWriteMergerNode::addDetails(std::stringstream& /* stream */) const {
+  // TODO Add connector details.
+}
+
+folly::dynamic TableWriteMergerNode::serialize() const {
+  auto obj = PlanNode::serialize();
+  VELOX_CHECK_EQ(sources_.size(), 1);
```

Comment:
NIT:
VELOX_CHECK_EQ(sources_.size(), 1, "TableWriteMergerNode source size must be 1");